blockchain/fullblocktests/generate.go

```diff
@@ -1575,11 +1576,11 @@ func Generate(includeLargeReorg bool) (tests [][]TestInstance, err error) {
 	// ---------------------------------------------------------------------
 
 	// Create a private/public key pair for signing transactions.
-	privKey, err := secp256k1.GeneratePrivateKey()
+	privKey, x, y, err := secp256k1.GenerateKey(rand.Reader)
```

Comment:
Is this leftover for various iterations?  I see now reason to change this.

Reply:
Alright, nevermind.  You need the raw bytes, so rather than calling serialize multiple times, you just grabbed the bytes here.